src/js/modules/FormFeedback.js

```diff
@@ -0,0 +1,231 @@
+      console.log('ERROR:', e);
+
+      this.setState({
+        loading: false,
+        error: true
+      })
+    })
+  }
+
+  handleReset = (e) => {
+    this.logEvent('reset');
+    this.setState({
+      step: 1,
+      loading: false,
+      emoji: null,
+      feedback: null,
+      error: null
+    })
+  }
+
+  render() {
+
+    const {intl} = this.props;
+
+    return (
+      <div className="coa-FormFeedback">
+
+      { this.state.step === 1 && (
+        <div className="coa-FormFeedback__step1">
+          <SectionHeaderSerif title={intl.formatMessage(i18nMessages.step1title)} />
+          <div className="coa-FormFeedback__emojis">
+            <button onClick={this.handleEmojiClick}
+              value="-2"
+              aria-label={intl.formatMessage(i18nMessages.emojiDisappointed)}
+            >
+              <img src={emojiDisappointed} alt={intl.formatMessage(i18nMessages.emojiDisappointed)} />
+            </button>
+            <button onClick={this.handleEmojiClick}
+              value="-1"
+              aria-label={intl.formatMessage(i18nMessages.emojiSad)}
+            >
+              <img src={emojiSad} alt={intl.formatMessage(i18nMessages.emojiSad)} />
+            </button>
+            <button onClick={this.handleEmojiClick}
+              value="0"
+              aria-label={intl.formatMessage(i18nMessages.emojiNeutral)}
+            >
+              <img src={emojiNeutral} alt={intl.formatMessage(i18nMessages.emojiNeutral)} />
+            </button>
+            <button onClick={this.handleEmojiClick}
+              value="1"
+              aria-label={intl.formatMessage(i18nMessages.emojiSlightlySmiling)}
+            >
+              <img src={emojiSlightlySmiling} alt={intl.formatMessage(i18nMessages.emojiSlightlySmiling)} />
+            </button>
+            <button onClick={this.handleEmojiClick}
+              value="2"
+              aria-label={intl.formatMessage(i18nMessages.emojiGrinning)}
+            >
+              <img src={emojiGrinning} alt={intl.formatMessage(i18nMessages.emojiGrinning)} />
+            </button>
+          </div>
+        </div>
+      )}
+
+      { this.state.step === 2 && (
+        <div className="coa-FormFeedback__step2">
+          <SectionHeaderSerif title={intl.formatMessage(i18nMessages.step2titlea)} />
+          <SectionHeaderSerif title={intl.formatMessage(i18nMessages.step2titleb)} />
+          <form>
+            { this.state.error && (
+              <p className="coa-FormFeedback__error">Oh no, something went wrong! Please, try submitting your feedback again.</p>
+            )}
+            <label htmlFor="site-feedback-textarea"
+              className="coa-sr-only"
+            >{intl.formatMessage(i18nMessages.step2titleb)}</label>
+            <textarea id="site-feedback-textarea"
+              onChange={this.handleTextAreaChange}></textarea>
+            <p>Please do not enter private information because your feedback will be public.</p>
```

Comment:
I know this is a content thing, but this struck me as odd. If I saw this messaging, I would hesitate to give feedback. Have you talked to them about that?